main.go

```diff
@@ -69,7 +89,13 @@ func main() {
 				nodeTaintedCount.Add(1)
 			}
 		})
-		fmt.Printf("%s,%d,%d,%d,%d,%d,%d,%d,%d,%d,%d,%d\n", time.Now().Format(time.RFC3339), len(nodeList.Items), nodeReadyCount.Load(), nodeTaintedCount.Load(), nodeDeletingCount.Load(), len(nodeClaimList.Items), launchedCount.Load(), registeredCount.Load(), initializedCount.Load(), driftedCount.Load(), disruptedCount.Load(), deletingCount.Load())
+		s := fmt.Sprintf("%s,%d,%d,%d,%d,%d,%d,%d,%d,%d,%d,%d,%d", time.Now().Format(time.TimeOnly), len(nodeList.Items), nodeReadyCount.Load(), nodeUnhealthyCount.Load(), nodeTaintedCount.Load(), nodeDeletingCount.Load(), len(nodeClaimList.Items), launchedCount.Load(), registeredCount.Load(), initializedCount.Load(), driftedCount.Load(), disruptedCount.Load(), deletingCount.Load())
+		err = writer.Write(strings.Split(s, ","))
```

Comment:
Can we consider using a multi-writer so that we can write to stdout and the file at exactly the same time?